CHANGELOG.md

```diff
@@ -1,13 +1,12 @@
 # Change log for Microsoft365DSC
 
+# UNRELEASED
+
+* EXOTransportConfig
+  * New resource for Exchange Online transport configuration.
+
 # 1.22.309.1
 
```

Comment:
I think this version of the changelog removes some lines from the version released yesterday.